MiddleWidgets/PageLyricList.cpp

```diff
@@ -592,7 +592,7 @@ void PageLyricList::OnAddToMakingHistory(QString song, QString lrc)
     LyricListItem item;
     item.song = song;
     item.lyric = lrc;
-    historyList.items.push_back(item);        //这里的更改直接对  listData 生效
+    historyList.items.insert(0,item);          //这里的更改直接对  listData 生效
```

Comment:
I think using [`void QVector::prepend(const T &value)`](https://doc.qt.io/qt-5/qvector.html#prepend-1) can make the code a bit simpler.

No modification is required, though.